server/metrics/metrics.go

```diff
@@ -37,6 +37,9 @@ const (
 	/// Cache event type: `hit`, `miss`, or `upload`.
 	CacheEventTypeLabel = "cache_event_type"
 
+	/// Cache name: Custom name to describe the cache, like "pebble-cache".
+	CacheNameLabel = "cache_name_label"
```

Comment:
remove the "_label" suffix from the name

Reply:
Done